packages/pwa/mixin.core.js

```diff
@@ -7,15 +7,40 @@ const ServiceWorkerPlugin = require('./lib/service-worker-plugin');
 const getAssetPath = (...args) => trimSlashes(join(...args));
 
 class PWAMixin extends Mixin {
-  configureBuild(webpackConfig, { allLoaderConfigs }, target) {
+  configureBuild(
+    webpackConfig,
+    { fileLoaderConfig, urlLoaderConfig, allLoaderConfigs },
+    target
+  ) {
+    const filename = getAssetPath(
+      this.config.assetPath,
+      '[name]-[hash:16].[ext]'
+    );
+    const emitFile = target === 'build' || target === 'develop';
+
+    // Rolling back to file-loader here, because the app-manifest-loader
+    // isn't able to deal with Asset Modules, yet.
+    delete fileLoaderConfig.type;
+    delete fileLoaderConfig.generator;
+    fileLoaderConfig.loader = require.resolve('file-loader');
+    fileLoaderConfig.options = {
+      name: filename,
+      esModule: false,
+      emitFile,
+    };
+    urlLoaderConfig.oneOf[0] = {
```

Comment:
Should we check here that `onefOf[0]` is actually the `noinline` loader / use `.find()` to find the correct one?